sdk/ai/Azure.AI.Projects/tests/Samples/Agent/Sample_Agent_Image_Inputs.cs

```diff
@@ -0,0 +1,236 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.Projects.Tests
+{
+    /// <summary>
+    /// Demonstrates examples of sending an image URL and image file (along with optional text)
+    /// as a structured content block in a single message.
+    /// </summary>
+    /// <remarks>
+    /// The examples shows how to create an agent, open a thread,
+    /// post content blocks combining text and image inputs,
+    /// and then run the agent to see how it interprets the multimedia input.
+    /// </remarks>
+    public partial class Sample_Agent_ContentBlocks : SamplesBase<AIProjectsTestEnvironment>
+    {
+        [Test]
+        [AsyncOnly]
+        public async Task ImageUrlInMessageExampleAsync()
+        {
+            #region Snippet:ImageUrlInMessageCreateClient
+#if SNIPPET
+            var connectionString = System.Environment.GetEnvironmentVariable("PROJECT_CONNECTION_STRING");
+            var modelDeploymentName = System.Environment.GetEnvironmentVariable("MODEL_DEPLOYMENT_NAME");
+#else
+            var connectionString = TestEnvironment.AzureAICONNECTIONSTRING;
+            var modelDeploymentName = TestEnvironment.MODELDEPLOYMENTNAME;
+#endif
+            // Create an AgentsClient, enabling agent-management and messaging.
+            AgentsClient client = new AgentsClient(connectionString, new DefaultAzureCredential());
```

Comment:
Nit: We can say 
```
AgentsClient client = new (connectionString, new DefaultAzureCredential());
```

Reply:
I'll keep the existing, in samples we seem to use both approaches.